frontend/src/metabase/collections/components/BaseTableItem.tsx

```diff
@@ -112,7 +124,7 @@ export function BaseTableItem({
               size={16}
               status={item.moderated_status}
             />
-            {item.description && (
+            {item.description?.trim() && (
```

Comment:
I think we shouldn't need to do this. We should trust that our inputs have been correctly sanitized. If they're not I think it's best to solve this problem elsewhere as we likely display the description in a dozen places.